src/PostgREST/CLI.hs

```diff
@@ -230,3 +246,227 @@ exampleConfigFile =
       |## When none is provided, 660 is applied by default
       |# server-unix-socket-mode = "660"
       |]
+exampleConfig ExampleDb =
+  [str|## Admin server used for checks. It's disabled by default unless a port is specified.
+      |# ALTER ROLE authenticator SET pgrst.admin_server_port = '3001';
+      |
+      |## The database role to use when no client authentication is provided
+      |# ALTER ROLE authenticator SET pgrst.db_anon_role = 'anon';
+      |
+      |## Notification channel for reloading the schema cache
+      |ALTER ROLE authenticator SET pgrst.db_channel = 'pgrst';
+      |
+      |## Enable or disable the notification channel
+      |ALTER ROLE authenticator SET pgrst.db_channel_enabled = 'true';
+      |
+      |## Enable in-database configuration
+      |ALTER ROLE authenticator SET pgrst.db_config = 'true';
+      |
+      |## Function for in-database configuration
+      |# ALTER ROLE authenticator SET pgrst.db_pre_config = 'postgrest.pre_config';
+      |
+      |## Extra schemas to add to the search_path of every request
+      |ALTER ROLE authenticator SET pgrst.db_extra_search_path = 'public';
+      |
+      |## Limit rows in response
+      |# ALTER ROLE authenticator SET pgrst.db_max_rows = '1000';
+      |
+      |## Allow getting the EXPLAIN plan through the `Accept: application/vnd.pgrst.plan` header
+      |# ALTER ROLE authenticator SET pgrst.db_plan_enabled = 'false';
+      |
+      |## Number of open connections in the pool
+      |ALTER ROLE authenticator SET pgrst.db_pool = '10';
+      |
+      |## Time in seconds to wait to acquire a slot from the connection pool
+      |# ALTER ROLE authenticator SET pgrst.db_pool_acquisition_timeout = '10';
+      |
+      |## Time in seconds after which to recycle pool connections
+      |# ALTER ROLE authenticator SET pgrst.db_pool_max_lifetime = '1800';
+      |
+      |## Time in seconds after which to recycle unused pool connections
+      |# ALTER ROLE authenticator SET pgrst.db_pool_max_idletime = '30';
+      |
+      |## Allow automatic database connection retrying
+      |# ALTER ROLE authenticator SET pgrst.db_pool_automatic_recovery = 'true';
```

Comment:
Oh, these don't work. The available in-db settings are the ones here https://github.com/PostgREST/postgrest/blob/85a4e35a2de869ddf9b6c5f6893a3154767cbc76/src/PostgREST/Config/Database.hs#L45-L71